doc/source/release-notes/Percona-Server-5.5.42-37.1.rst

```diff
@@ -0,0 +1,22 @@
+.. rn:: 5.5.42-37.1
+
+==============================
+ |Percona Server| 5.5.42-37.1
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.42-37.1 on March 4th, 2015. Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.42-37.1/>`_ and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.42 <http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-42.html>`_, including all the bug fixes in it, |Percona Server| 5.5.42-37.1 is now the current stable release in the 5.5 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.5.42-37.1 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.5.42-37.1>`_. 
+
+Bugs Fixed
+==========
+
+ ``RPM`` pre-install script assumed that the ``PID`` file was always located in the :variable:`datadir`. As a result, during installation, wrong assumption could be made if the server was running or not. Bug fixed :bug:`1201896`.
+ 
+ ``SHOW GRANTS`` displayed only the privileges granted explicitly to the named account. Other privileges which might be available to the account, (if an anonymous account exists, the named account might be able to use its privileges) were not displayed. Bug fixed :bug:`1354988` (upstream :mysqlbug:`53645`).
+ 
+ |InnoDB| lock monitor output was printed even if it was not requested. Bug fixed :bug:`1418996`.
+
+ The stored procedure key (``# Stored procedure:``) was using spaces instead of underscores in the :ref:`slow_extended_55`. Bug fixed :bug:`1419230`.
```

Comment:
The stored procedure key was made consistent with other keys in the :ref:... by replacing space with an underscore